### ISSUE TITLE
[BUG] Different font styles in Chromium in Playwright and Pupperteer.

### ISSUE THREAD

**Issue by volekerb** [microsoft/playwright#870]
**Context:**
- Playwright Version: 0.10.0
- Operating System: Mac
- Extra: 

**Code Snippet**
There are several code snippets: on Puppeteer and Playwright. So decided to give a link to my repo. 
https://github.com/volekerb/PlayWrightExamples
There I've pushed diff image: https://github.com/volekerb/PlayWrightExamples/blob/master/diff_image.png

**Describe the bug**
Screenshot taken by Playwright on Chromium with default setting uses a bit different fonts compare to Puppeteer with Chromium.

Puppeteer version: ^1.17.0

**Note**
I guess this not the highest priority but could be important for people who takes care about pixel perfect and will try to move to playwright in one shoot.

**Comment by pavelfeldman** [comment]
[1.17.0](https://github.com/puppeteer/puppeteer/releases/tag/v1.17.0) was using Chromium **76.0.3803.0** from April 2019. Playwright 0.10.0 is using **81.0.4044**, scheduled to go stable mid March 2020. There is almost a year of engine changes between the two. So I think changes to the rendered image are expected.

**Comment by pavelfeldman** [comment]
(Please feel free to reopen if you think there is a font that the two treated differently, but we don't really do anything special about the fonts, so I don't think there is any regression on the Playwright side)

**Comment by volekerb** [comment]
Sure, thank you! Just wanted to make sure you're aware of such behavior.

**Comment by sohaib17** [comment]
While working on code migration I hit a blocker because of this nuance. I noticed that screenshots made by Puppeteer are different than the ones made by Playwright. Upon debugging it turned out that Playwright was somehow using different font to render the screenshot.

I thought that it is expected since browser version is different but to my surprise problem persisted even when I pointed Playwright to use the same binary (bundled with Puppeteer) using [executable_path](https://playwright.dev/python/docs/api/class-browsertype#browser-type-launch-persistent-context-option-executable-path) argument.

My project performs image similarity analysis that is why change in font is turning out to be a blocker in migration. Any idea to bypass/restore Puppeteer like behavior would be helpful.

This [URL](https://onedrive.live.com/?authkey=%21ACkeIHsdSDtBmVN13YQ&cid=5087FB9A0D34D4E3&id=5087FB9A0D34D4E3%21115&parId=5087FB9A0D34D4E3%21104&action=locate) can be used to see the difference. Attaching HTMLs and PNGs as a reference.

[playwright.zip](https://github.com/microsoft/playwright/files/7196118/playwright.zip)
[puppeteer.zip](https://github.com/microsoft/playwright/files/7196127/puppeteer.zip)

OS: Ubuntu 18.04
Puppeteer: 5.3.1 (Chromium 86.0.4240.0)
Playwright: 1.14.1 (Python) (Chromium 94.0.4595.0)
